diff --git a/README.txt b/README.txt
index d43d79e..95472da 100644
--- a/README.txt
+++ b/README.txt
@@ -5,7 +5,7 @@ Imports mini-panels, pages and page variants from corresponding subfolders.
 
 INSTALLATION
 -----------
-1. Create 'sites/all/imports' folder.
+1. Create panels import directory: 'sites/all/imports' by default.
 2. Create all necessary subfolders for panels:
 	-> sites/all/imports/page_manager/handlers
 	-> sites/all/imports/page_manager/pages
@@ -16,6 +16,7 @@ INSTALLATION
 4. Do not forget to add '<?php' line to the top of the files.
 5. Move files to corresponding subfolders.
 6. Install this module.
+7. Go to 'admin/config/content/panels-import' and configure the import directory.
 
 MAIN IDEA
 -----------
diff --git a/panels_import.admin.inc b/panels_import.admin.inc
new file mode 100644
index 0000000..32296d4
--- /dev/null
+++ b/panels_import.admin.inc
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
+ * Administrative page callbacks for the panels_import module.
+ */
+
+/**
+ * Form builder to configure panels_import behaviors.
+ *
+ * @ingroup forms
+ */
+function panels_import_settings() {
+
+  $form['panels_import_directory'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Import directory'),
+    '#default_value' => variable_get('panels_import_directory', 'sites/all/imports'),
+    '#description' => t('This directory holds all of your import files.'),
+    '#required' => TRUE,
+  );
+
+  $form['#validate'][]  = 'panels_import_settings_validate';
+
+  return system_settings_form($form);
+}
+
+/**
+ * Form validation handler for panels_import_settings().
+ *
+ * @param $form
+ * @param $form_state
+ */
+function panels_import_settings_validate($form, &$form_state) {
+  if (!is_dir($form_state['values']['panels_import_directory'])) {
+    form_set_error('panels_import_directory', t('That directory does not exist, please create it.'));
+  }
+}
\ No newline at end of file
diff --git a/panels_import.info b/panels_import.info
index f56dcfb..2e856ef 100644
--- a/panels_import.info
+++ b/panels_import.info
@@ -2,6 +2,7 @@ name = Panels Import
 description = Imports mini-panels, pages and page variants from corresponding subfolders.
 core = 7.x
 package = "Panels"
+configure = admin/structure/taxonomy
 
 dependencies[] = page_manager
 dependencies[] = panels
diff --git a/panels_import.install b/panels_import.install
index 928b516..fe9a36b 100644
--- a/panels_import.install
+++ b/panels_import.install
@@ -9,7 +9,8 @@
  */
 function panels_import_install() {
   db_update('system')->fields(array('weight' => 99))->condition('name', 'panels_import')->execute();
-  drupal_set_message(t("The Panels Import Module has been installed."));
+  variable_set('panels_import_directory', 'sites/all/imports');
+  drupal_set_message(t("The Panels Import Module has been installed and using sites/all/imports for panel import included files. This directory can be configured."));
 }
 
 /**
@@ -19,20 +20,3 @@ function panels_import_uninstall() {
   drupal_set_message(t("The Panels Import Module has been uninstalled."));
 }
 
-/**
- * Implements hook_requirements().
- */
-function panels_import_requirements($phase) {
-  $directory = 'sites/all/imports';
-  $requirements = array();
-
-  if (!is_dir($directory)) {
-    $requirements['panels_import'] = array(
-      'title' => t('Panels Import'),
-      'value' => t('The directory %directory does not exist!', array('%directory' => $directory)),
-      'severity' => REQUIREMENT_ERROR,
-    );
-  }
-
-  return $requirements;
-}
diff --git a/panels_import.module b/panels_import.module
index 4a293fd..ea18d45 100644
--- a/panels_import.module
+++ b/panels_import.module
@@ -17,6 +17,24 @@ function panels_import_ctools_plugin_api($module, $api) {
 }
 
 /**
+ * Implements hook_menu().
+ */
+function panels_import_menu() {
+
+  $items['admin/config/content/panels-import'] = array(
+    'title' => 'Panels Import Settings',
+    'description' => 'Manage settings of panels import.',
+    'page callback' => 'drupal_get_form',
+    'page arguments' => array('panels_import_settings'),
+    'access arguments' => array('administer site configuration'),
+    'file' => 'panels_import.admin.inc',
+    'type' => MENU_NORMAL_ITEM,
+  );
+
+  return $items;
+}
+
+/**
  * Implementation of hook_pm_existing_pages_info()
  */
 function panels_import_pm_existing_pages_info() {
@@ -38,7 +56,12 @@ function panels_import_pm_existing_pages_info() {
  *   Exported object or NULL.
  */
 function _panels_import_load_defaults($module, $subfolder, $variable) {
-  $imports_dir = 'sites/all/imports';
+  $imports_dir = variable_get('panels_import_directory', 'sites/all/imports');
+
+  if (!is_dir($imports_dir)) {
+    drupal_set_message(t('The %dir directory does not exist.', array('%dir', $imports_dir)), 'error');
+    return NULL;
+  }
 
   $folder = "$imports_dir/$module/$subfolder";
   $files = file_scan_directory($folder, '/\.inc$/', array('recurse' => FALSE));
